crawlfrontier/settings/default_settings.py

```diff
@@ -6,9 +6,9 @@
 REQUEST_MODEL = 'crawlfrontier.core.models.Request'
 RESPONSE_MODEL = 'crawlfrontier.core.models.Response'
 MIDDLEWARES = [
-    'crawlfrontier.contrib.middlewares.domain.DomainMiddleware',
+    # 'crawlfrontier.contrib.middlewares.domain.DomainMiddleware',
```

Comment:
IMO, it doesn't make sense to leave comments there. One could easily find them in docs or in sources. So please feel free to remove them.